micro_sam/training/trainable_sam.py

```diff
@@ -67,9 +69,6 @@ def forward(
             batched_inputs: The batched input images and prompts.
             image_embeddings: The precompute image embeddings. If not passed then they will be computed.
             multimask_output: Whether to predict mutiple or just a single mask.
-
-        Returns:
```

Comment:
Shouldn't we leave a `Returns` here?

Reply:
This is quite important. I missed it. Thanks!